dart/lib/src/protocol/sentry_span.dart

```diff
@@ -16,6 +13,9 @@ class SentrySpan extends ISentrySpan {
   Map<String, List<MetricSummary>>? _metricSummaries;
   late final DateTime _startTimestamp;
   final Hub _hub;
+  bool _isRootSpan = false;
```

Comment:
the reason why we add this is because `SentryTracer` is just a wrapper around a `SentrySpan`.

we need this flag so we can call `setMeasurement` only on the root span